Sources/Clappr/Classes/Base/Loader.swift

```diff
@@ -1,11 +1,11 @@
 open class Loader {
     internal(set) open var playbackPlugins: [Plugin.Type] = [AVFoundationPlayback.self, NoOpPlayback.self]
     #if os (iOS)
-    internal(set) open var containerPlugins: [Plugin.Type] = [PosterPlugin.self, LoadingContainerPlugin.self]
+    internal(set) open var containerPlugins: [Plugin.Type] = [PosterPlugin.self]
     #else
     internal(set) open var containerPlugins: [Plugin.Type] = []
     #endif
-    internal(set) open var corePlugins = [Plugin.Type]()
+    internal(set) open var corePlugins: [Plugin.Type] = [LoadingCorePlugin.self]
```

Comment:
Hey @CristianDRM , remember to set it only for iOS. In tvOS, we do not need this plugin.